Issue866 adding network schema

DSuveges opened the pull request:
This branch is for adding the json schema describing the network data provided by collaborators or generated by ourselves. This schema file will help to maintain the standards required for importing and generating these files.

**Important:** to ensure we can use the schema with the object wrapper for [JSON Schema definitions](https://pypi.org/project/python-jsonschema-objects/0.0.13/) the schema is written following the json schema [draft-4](https://json-schema.org/specification-links.html#draft-4) guidelines.

DSuveges commented:
> The schema itself looks good to me but I wonder if having in the same repo an schema following draft-4 (this one) and another one following draft-7 (the main ot schema) may be inconsistent and could cause issues. Also, can this version be used directly with the validator?

I think it can be used with the validator, but I need to check. Your points are valid, I'm not 100% convinced if this is the best place, but I didn't find a more suitable place to host this file so far. I'm open to alternatives.

AsierGonzalez commented:
One way to avoid confusion could be to move the draft-4 schema to a sybdirectory in the repository called `draft-4`.

DSuveges commented:
Changes since the previous review:
* Changes in schema describing the network is updated.
* Schema file is moved into a folder called `draft4_schemas` indicating the compatible schema draft version.
* The schema describing the evidences following the draft-4 specification is also moved into this folder.
